.github/workflows/sv-tests-ci.yml

```diff
@@ -218,3 +218,22 @@ jobs:
         uses: geekyeggo/delete-artifact@v1
         with:
           name: ${{ steps.get-artifacts-to-delete.outputs.artifacts }}
+  Automerge:
```

Comment:
If I'm not missing anything, this will also run on push to ``master`` branch or on schedule. I'm not sure what ``github.actior`` is set then, but it will probably be better to limit it only to PR.

Reply:
Good point, I'm not sure either. Let's add an explicit check